test/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/ReliefDetailsAnswersSpec.scala

```diff
@@ -66,7 +66,7 @@ class ReliefDetailsAnswersSpec extends WordSpec with Matchers with ScalaCheckDri
 
       }
 
-      "have a method which unsets values of private residents relief and letting relief" in {
+      "have a method which unsets values of private residence relief and letting relief" in {
```

Comment:
whoops, I've been typing it wrong this whole time!

Reply:
I know the prototype had it wrong in some places too for a while and still does in some code refs - thought it best to avoid confusion :) if you think about it, it's talking about the property being a private residence not a relief for residents who are private lol